app/org/sagebionetworks/bridge/models/schedules/ActivityScheduler.java

```diff
@@ -115,7 +130,7 @@ protected DateTime getFirstEventDateTime(ScheduleContext context, String eventId
             String[] eventIds = eventIdsString.trim().split("\\s*,\\s*");
             for (String thisEventId : eventIds) {
                 if (context.getEvent(thisEventId) != null) {
-                    eventDateTime = context.getEvent(thisEventId).withZone(context.getZone());
+                    eventDateTime = context.getEvent(thisEventId);
```

Comment:
Are we guaranteed that events in the context are always in UTC? Do we want to force UTC here as well?